README.md

```
@@ -1 +1,309 @@
+  value = module.servicebus.debug_locals
+}
+
+output "vnet_links_info" {
+  value = module.servicebus.private_dns_zone_vnet_links_info
+}
+```
+
+### 2.5. examples
+
+#### Variable requirement by `network_mode`
+| `network_mode`       | `subnet_ids`              | `vnet_ids` | 
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)